src/client/pages/Jeugd/Jeugd-render-config.ts

```diff
@@ -0,0 +1,33 @@
+import { routeConfig, themaId, themaTitle } from './Jeugd-thema-config';
+import { JeugdDetail } from './JeugdDetail';
+import { JeugdThemaPagina } from './JeugdThema';
+import { FeatureToggle } from '../../../universal/config/feature-toggles';
+import { isLoading } from '../../../universal/helpers/api';
+import { AppState } from '../../../universal/types';
+import { IconJeugd } from '../../assets/icons';
+import { ThemaMenuItem } from '../../config/thema-types';
+
+export const JeugdRoutes = [
+  {
+    route: routeConfig.themaPage.path,
+    Component: JeugdThemaPagina,
+    isActive: FeatureToggle.zorgnedLeerlingenvervoerActive,
+  },
+  {
+    route: routeConfig.detailPage.path,
+    Component: JeugdDetail,
+    isActive: FeatureToggle.zorgnedLeerlingenvervoerActive,
+  },
+];
+
+export const menuItem: ThemaMenuItem = {
+  title: themaTitle,
+  id: themaId.JEUGD,
+  profileTypes: ['private'],
+  isActive: (appState: AppState) =>
+    FeatureToggle.zorgnedLeerlingenvervoerActive &&
+    !isLoading(appState.JEUGD) &&
+    !!appState.JEUGD.content?.length,
```

Comment:
[nitpick] Is 't zo niet iets beter leesbaar? Ik vind die arrowfunctions soms wat lastiger te lezen.

```suggestion
  isActive(appState: AppState) {
    return FeatureToggle.zorgnedLeerlingenvervoerActive && 
      !isLoading(appState.JEUGD) && 
      !!appState.JEUGD.content?.length;
  }
```

Reply:
Geen idee eigenlijk, ben nu die arrows gewent, maar ik zal het wel voortaan zo doen, toch een win als het voor je leesbaarder word